vnext/Microsoft.ReactNative/Fabric/Composition/Composition.Input.h

```diff
@@ -28,4 +33,123 @@ struct KeyRoutedEventArgs : winrt::implements<
+
+  PointerPointProperties(
+      bool isBarrelButtonPressed,
+      bool isCanceled,
+      bool isEraser,
+      bool isHorizontalMouseWheel,
+      bool isInRange,
+      bool isInverted,
+      bool isLeftButtonPressed,
+      bool isMiddleButtonPressed,
+      bool isPrimary,
+      bool isRightButtonPressed,
+      bool isXButton1Pressed,
+      bool isXButton2Pressed,
+      bool touchConfidence,
+      int32_t mouseWheelDelta,
+      float orientation,
+      PointerUpdateKind pointerUpdateKind,
+      float pressure,
+      float twist,
+      float xTilt,
+      float yTilt);
+
+  winrt::Windows::Foundation::Rect ContactRect() noexcept;
+  bool IsBarrelButtonPressed() noexcept;
+  bool IsCanceled() noexcept;
+  bool IsEraser() noexcept;
+  bool IsHorizontalMouseWheel() noexcept;
+  bool IsInRange() noexcept;
+  bool IsInverted() noexcept;
+  bool IsLeftButtonPressed() noexcept;
+  bool IsMiddleButtonPressed() noexcept;
+  bool IsPrimary() noexcept;
+  bool IsRightButtonPressed() noexcept;
+  bool IsXButton1Pressed() noexcept;
+  bool IsXButton2Pressed() noexcept;
+  int32_t MouseWheelDelta() noexcept;
+  float Orientation() noexcept;
+  PointerUpdateKind PointerUpdateKind() noexcept;
+  float Pressure() noexcept;
+  bool TouchConfidence() noexcept;
+  float Twist() noexcept;
+  float XTilt() noexcept;
+  float YTilt() noexcept;
+
+ private:
+  winrt::Windows::UI::Input::PointerPointProperties m_systemPPP{nullptr};
+
+  // When not using m_systemPPP
+  bool m_isBarrelButtonPressed : 1;
+  bool m_isCanceled : 1;
+  bool m_isEraser : 1;
+  bool m_isHorizontalMouseWheel : 1;
+  bool m_isInRange : 1;
+  bool m_isInverted : 1;
+  bool m_isLeftButtonPressed : 1;
+  bool m_isMiddleButtonPressed : 1;
+  bool m_isPrimary : 1;
+  bool m_isRightButtonPressed : 1;
+  bool m_isXButton1Pressed : 1;
+  bool m_isXButton2Pressed : 1;
+  bool m_touchConfidence : 1;
+  int32_t m_mouseWheelDelta;
+  float m_orientation;
+  winrt::Microsoft::ReactNative::Composition::Input::PointerUpdateKind m_pointerUpdateKind;
+  float m_pressure;
+  float m_twist;
+  float m_xTilt;
+  float m_yTilt;
+};
+
+struct PointerPoint : PointerPointT<PointerPoint> {
+  PointerPoint(const winrt::Windows::UI::Input::PointerPoint &pp);
+  PointerPoint(uint32_t msg, uint64_t wParam, int64_t lParam);
+
+  uint32_t FrameId() noexcept;
+  bool IsInContact() noexcept;
+  PointerDeviceType PointerDeviceType() noexcept;
+  uint32_t PointerId() noexcept;
+  winrt::Windows::Foundation::Point Position() noexcept;
+  winrt::Microsoft::ReactNative::Composition::Input::PointerPointProperties Properties() noexcept;
+  uint64_t Timestamp() noexcept;
+  winrt::Microsoft::ReactNative::Composition::Input::PointerPoint GetTransformedPoint(
+      const IPointerPointTransform &transform) noexcept;
+
+ private:
+  bool IsPointerMessage(uint32_t message) noexcept;
+
+  // Windows::Input
+  winrt::Windows::UI::Input::PointerPoint m_systemPP{nullptr};
```

Comment:
nit: maybe rename to something less abbreviated?

```suggestion
  winrt::Windows::UI::Input::PointerPoint m_sysPointerPoint{nullptr};
```